packages/code-du-travail-frontend/pages/_error.js

```diff
@@ -1,71 +1,47 @@
-import React from "react";
-import Link from "next/link";
+// https://nextjs.org/docs/advanced-features/custom-error-page
+import React, { useEffect } from "react";
 import styled from "styled-components";
-import { Section, theme } from "@socialgouv/react-ui";
+import { Button, Container, Section } from "@socialgouv/react-ui";
 import { Layout } from "../src/layout/Layout";
 import { initializeSentry, notifySentry } from "../src/sentry";
 
 initializeSentry();
 
-export default class Error extends React.Component {
-  static async getInitialProps({ res, err }) {
-    const statusCode = res ? res.statusCode : err ? err.statusCode : null;
-    return { statusCode, message: err.message };
-  }
-
-  componentDidMount() {
-    const { statusCode, message } = this.props;
+export default function CustomError({ message, statusCode }) {
+  useEffect(() => {
     if (statusCode && statusCode >= 400) {
       notifySentry(statusCode, message);
     }
-  }
-
-  render() {
-    const { statusCode } = this.props;
-    return (
-      <Layout>
-        <Section>
-          <FlexCenterer>
-            <P>
-              {this.props.statusCode
-                ? `Une erreur ${statusCode} est apparue sur le serveur :/`
-                : "Une erreur est apparue sur le client"}
-              <br />
-            </P>
-            <Smaller>Notre équipe technique a été informée.</Smaller>
-            <P>
-              <Link href="/">
-                <a>Retour à la page d’accueil</a>
-              </Link>
-            </P>
-          </FlexCenterer>
-        </Section>
-      </Layout>
-    );
-  }
+  }, [message, statusCode]);
+
+  const content = (
+    <>
+      <h1>
+        {statusCode
+          ? `Nous sommes navrés, une erreur ${statusCode} est survenue.`
```

Comment:
pas très fan de navrés, je mettrais peut être un truc comme 
"Nous ne sommes pas en mesure de vous afficher cette page suite à une erreur."
si tu peux faire comme la page 404 et mettre le code en haut c'est bien

Reply:
j'ai mis le code en haut comme sur le 404. Par contre pour le message on etait plutôt d'accord avec Virginie. Personnellement je suis adepte de ce concept: "Over time, a customer service theorem emerged. When a customer brings a complaint, there are two tokens on the table: It’s Nothing and It’s Awful." Plus on minimisera l'erreur, plus elle sera perçue comme importante par l'utilisateur. Dire que l'on est navré, ce qui est vrai en ce qui me concerne, tend à faire pencher le token du côté du "it's nothing" https://mondaynote.com/united-broken-culture-6b35267c8a10

Reply:
Nous sommes désolés ?

Mettre un message type "notre équipe a été informée et interviendra..." permet aussi de rassurer :)

Reply:
"Désolé, une erreur s'est produite..."
"Notre équipe a été informée et interviendra sous peu." 

Vous en pensez quoi ?

Reply:
en fait je n'aime pas démarrer une phrase avec désolé, j'aurais tendance à d'abord expliquer ce qu'il se passe (une erreur s'est produite, on ne peut pas afficher la page" et m'excuser en suite  et effectivement expliquer les mesures prises. (nos équipes ont été informés)

Reply:
Du genre "Une erreur s'est produite, nous somme désolé" ?

Reply:
(J'inverse l'ordre de la phrase)

Reply:
oui mais c'est du nitpick, si ca te saoule, on peut laisser comme c'est, ca va aussi

Reply:
Pour le coup "on ne peut pas afficher la page" ça ne me semble pas necessaire tant c'est intrinsèque à l'affichage d'une page d'erreur, et le désolé ne me choque gène vraiment pas là ou il est ! On va faire de l'AB testing sur la page pour voir ce que l'user préfère 🤣